src/services/fluxService.js

```diff
@@ -1288,6 +1289,73 @@ async function start() {
     }, 5 * 60 * 1000);
   }
 }
+
+async function getAllConnections (req, res) {
```

Comment:
Why should this be server fetching the data and then swnding the data to user. In the scenario coded it should be the user asking directly rather than proxying through server, lots of traffic that server would need to do.
Suggestions :
a) move it to client based solution
b) different approach where our server continuously scans the entite network and stores connections to db. Thus mat however be lots of data to store